application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -102,6 +122,34 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
             .insert());
     }
 
+    private void handleListCommand(SlashCommandInteractionEvent event) {
+        BiFunction<Long, Instant, String> getDescription = (channelId, remindAt) -> """
+                Channel: %s
+                Remind at: %s""".formatted(MessageUtils.mentionChannelById(channelId),
+                TimeFormat.DEFAULT.format(remindAt));
+
+        EmbedBuilder remindersEmbed = new EmbedBuilder().setTitle("Pending reminders")
+            .setColor(RemindRoutine.AMBIENT_COLOR);
+
+        long guildId = event.getGuild().getIdLong();
+        long userId = event.getUser().getIdLong();
+
+        database
+            .read(context -> context.selectFrom(PENDING_REMINDERS)
+                .where(PENDING_REMINDERS.GUILD_ID.eq(guildId)
+                    .and(PENDING_REMINDERS.AUTHOR_ID.eq(userId)))
+                .orderBy(PENDING_REMINDERS.CREATED_AT.asc())
+                .fetch())
+            .forEach(reminder -> remindersEmbed.addField(reminder.getContent(),
+                    getDescription.apply(reminder.getChannelId(), reminder.getRemindAt()), false));
+
+        if (remindersEmbed.getFields().isEmpty()) {
```

Comment:
this condition feels a bit odd. i would instead have expected that you check on the size of the returned DB list. like `if (pendingReminders.isEmpty())`, i think that would make it more readable.